hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/om/helpers/SnapshotInfo.java

```diff
@@ -714,4 +714,16 @@ public SnapshotInfo copyObject() {
         .setDeepCleanedDeletedDir(deepCleanedDeletedDir)
         .build();
   }
+
+  @Override
+  public String toString() {
+    return String.format("snapshotId: '%s', name: '%s', volumeName: '%s', bucketName: '%s', " +
+            "snapshotStatus: '%s', creationTime : '%s', deletionTime : '%s', pathPreviousSnapshotId : '%s', " +
+            "globalPreviousSnapshotId: '%s', snapshotPath: '%s', checkpointDir: '%s', dbTxSequenceNumber: '%s', " +
+            "deepClean: '%s', sstFiltered: '%s', referencedSize: '%s', referencedReplicatedSize: '%s', " +
+            "exclusiveSize: '%s', exclusiveReplicatedSize: '%s', deepCleanedDeletedDir: '%s'.",
+        snapshotId, name, volumeName, bucketName, snapshotStatus, creationTime, deletionTime, pathPreviousSnapshotId,
+        globalPreviousSnapshotId, snapshotPath, checkpointDir, dbTxSequenceNumber, deepClean, sstFiltered,
+        referencedSize, referencedReplicatedSize, exclusiveSize, exclusiveReplicatedSize, deepCleanedDeletedDir);
```

Comment:
nit: Please use simple string concatenation with one property per line.  Makes it easier to spot any misplaced items, as well as update in the future (add/remove items).

Example:

https://github.com/apache/ozone/blob/e2a4609f3ce58873a31cad832ad045f6d2938139/hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/om/helpers/OmKeyInfo.java#L395-L411

Reply:
Changed as suggested.

Reply:
Thanks!